### ISSUE TITLE
Why Dilbert can't ACL

### ISSUE THREAD

**Issue by brunokim** [jhuangtw/xg2xg#142]
There was an excellent doc about capability vs ACL-based permissions, called "Why Dilbert can't ACL", but I can't find any equivalent blog post outside. I also don't know who was the author.

Does anybody have a pointer to the contents of this? :)

**Comment by mmckenziedev** [comment]
@brunokim You'll probably have better luck asking in the group https://www.facebook.com/groups/exfaceeng

**Comment by brunokim** [comment]
@mmckenziedev Ex...Facebook engineers? Isn't this a xoogler group?

**Comment by mmckenziedev** [comment]
@brunokim You are correct. My mistake. There is Github group for people from Facebook similar to this. I clearly got the two of them confused.
